pkg/sanity/sanity.go

```diff
@@ -190,7 +190,7 @@ func NewTestConfig() TestConfig {
 		StagingPath:          os.TempDir() + "/csi-staging",
 		CreatePathCmdTimeout: 10 * time.Second,
 		RemovePathCmdTimeout: 10 * time.Second,
-		TestVolumeSize:       10 * 1024 * 1024 * 1024, // 10 GiB
+		TestVolumeSize:       1 * 1024 * 1024 * 1024, // 10 GiB
```

Comment:
Comment also needs to be updated.

Reply:
I had not enough space on my k8s cluster and didn't find flag to set test volume size at that time, that's why I changed it. But this change is not intended to be in final patch and I finally found `--csi.testvolumesize` flag.
It is rolled back now.